examples/tls_server/gen-ec.py

```diff
@@ -0,0 +1,39 @@
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.backends import default_backend
+from cryptography import x509
+from cryptography.x509.oid import NameOID
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import ec
+import datetime
+
+ec_key = ec.generate_private_key(ec.SECP256K1, default_backend())
+
+subject = issuer = x509.Name([
+    x509.NameAttribute(NameOID.COMMON_NAME, u"hendrix-tls-example"),
+ ])
+cert = x509.CertificateBuilder().subject_name(
```

Comment:
That's quite a method chain! Is there no way to do this with kwargs or a dict?  If not, using a short variable may be somewhat more readable ( and less javascriptic :-) ).

Reply:
Tell me about it dude.  We can change it if you want, but my intention (since this is an example) was to keep it as close as was reasonable to the tutorial snippet from cryptography.io:

https://cryptography.io/en/latest/x509/tutorial/#creating-a-self-signed-certificate